yodeploy/hooks/django.py

```diff
@@ -12,6 +13,19 @@
 log = logging.getLogger(__name__)
 
 
+def parse_django_version(version):
```

Comment:
I wasn't sure if it was safe to assume that `setuptools` would be installed.  Is that guaranteed?